timemachine/fe/single_topology_v3.py

```diff
@@ -577,15 +577,10 @@ def combine_masses(self):
         """
         Combine masses between two end-states using linear interpolation.
 
-        Parameters
-        ----------
-        lamb: float
-            Lambda value
-
         Returns
         -------
-        np.array
-            shape of self.get_num_atoms()
+        masses: list
```

Comment:
nit: `list of float`?

Reply:
Addressed in https://github.com/proteneer/timemachine/pull/846/commits/026cde2d361c148b04c80ece1b6491a3ae605db8